lib/pure/strmisc.nim

```diff
@@ -116,63 +116,44 @@ func parseFloatThousandSep*(str: string; sep = ','; decimalDot = '.'): float {.s
     doAssert parseFloatThousandSep("10.000,0", '.', ',') == 10000.0
     doAssert parseFloatThousandSep("1'000'000,000", '\'', ',') == 1000000.0
   assert sep != '-' and decimalDot notin {'-', ' '} and sep != decimalDot
+
+  proc raiseError(i: int, c: char) {.noinline, noreturn.} =
```

Comment:
`str` must be reported (c can be derived from i + str, if you want to also show `c`), otherwise  user won't have enough context to understand what triggered the exception in the general case where str isn't some litteral but is some arbitrary value